api/app/references/api.py

```diff
@@ -105,3 +114,41 @@ def get(self):
 Kind regards,
 The {event_description} team.
 """
+
+
+class ReferenceAPI(ReferenceMixin, restful.Resource):
+
+    @marshal_with(reference_fields)
+    @auth_required
+    def get(self):
+        LOGGER.debug('Received get request for reference')
+        user = user_repository.get_by_id(g.current_user['id'])
+        if not (user or user.is_admin):
```

Comment:
The `is_admin` flag indicates the user is a system admin. To determine if a user is an event admin, use the `is_event_admin` method instead (you'll also need an event_id parameter from the request)